app/views/tmdb/actor_more.html.erb

```diff
@@ -14,7 +14,7 @@
     <div class="credits">
       <%= render partial: "movie_credits", locals: { person_profile: @data.actor.profile, movie_credits: @data.actor.movie_credits } %>
 
-      <%= render partial: "seen_movie_credits", locals: { person_profile: @data.actor.profile, movies_seen: @data.movies_seen } %>
```

Comment:
These details are folded into the credits data now so this extra local variable is not needed